tensorflow_datasets/core/dataset_builder.py

```diff
@@ -1071,22 +1148,31 @@ def _generate_examples(self, **kwargs):
     """
     raise NotImplementedError()
 
-  def _download_and_prepare(self, dl_manager, download_config):
+  # pylint: disable=arguments-differ
```

Comment:
Always reactivate `pylint: enable` after using `pylint: enable`.

Also could you provide more context why this is required ?

Reply:
enabled it. It is required because this is the extended class and this overriden method has different params than default `_download_and_prepare` method.

Reply:
I see, rather than putting it on an empty line, it is best to add the comment on the same line it is used.

Reply:
Yes done